site/content/api/Command-And-Control-API.md

```diff
@@ -26,7 +26,9 @@ The following API defines operations that can be used by *Business Applications*
 
 1. *Business Application* has established an AMQP connection with the AMQP 1.0 Network.  
 2. *Business Application* has established an AMQP link in role *sender* with the target address `control/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to send messages to and `${tenant_id}` is the ID of the tenant that the device belongs to.
-3. *Business Application* has established an AMQP link in role *receiver* with the source address `control/${tenant_id}/${device_id}/${reply-to}`. Where `${reply-to}` may be any arbitrary string chosen by the client. This link is used by the client to receive the response of the executed command. This link’s source address is also referred to as the reply-to address for the request messages.
+3. *Business Application* has established an AMQP link in role *receiver* with the source address `control/${tenant_id}/${reply-id}`. Where `${reply-id}` may be any arbitrary string chosen by the client. This link is used by the client to receive the response of the executed command. This link’s source address is also referred to as the reply-to address for the request messages. 
+
+In cases where the response should be scoped for a device (one response link per device) we suggest to add the device-id as the first part of the reply-id (`control/${tenant_id}/${device_id}/..`). This allows the adapters to minimize the information needed correlate the requests of the device with the responses, since this information (`${device_id}`) is already known and does not need to be encoded in such an request-id.
```

Comment:
can we rephrase this to not use fourth person (`we suggest`)? we are not using this style anywhere in the API specs.
IMHO we also should not hint in the spec at the optimization that an adapter might or might not perform. However, it would be nice to have an example of a tenant scoped *reply-id* vs. a device scoped *reply-id* ...